4funccalc.php

```diff
@@ -0,0 +1,50 @@
+<html>
+	<body>
+		<h1> Four Function Calculator </h1> <br> <br>
+			<form action=""
+				<p><input type="text" name="variable1"/></p>
+				<p>
+						<select name="math" id="math">
+							<option value='0'>--Select function--</option>
+							<option value='add'> + </option>
+							<option value='sub'> - </option>
+							<option value='mult'> * </option>
+							<option value='div'> / </option>
+						</select>
+				</p>
+				<p><input type='text' name='variable2'/></p>
+				<input type= 'submit'/>
+			</form>
+	</body>
+</html>
```

Comment:
This is *super* minor, but your html tag ends here before the "The answer is" stuff is printed below. It'll still work, but you're support to put `</body></html>` at the very bottom.